src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -2033,4 +2089,30 @@ public function get_transaction_url( $order ) {
 		return parent::get_transaction_url( $order );
 	}
 
+    /**
+     * Isolates static addNotice calls.
+     *
+     * @param  string $message
+     * @param  string $type
+     */
+    protected function staticAddNotice($message, $type)
+    {
+        Mollie_WC_Plugin::addNotice(
+            __(
+                $message,
+                $type
+            )
+        );
+    }
+
+    /**
+     * Isolates static debug calls.
+     *
+     * @param  string $debugLine
+     */
+    protected function staticDebug($debugLine)
```

Comment:
Why do you prefix the method `static`? Ok you're calling a static method inside but that's an information that should be hidden to the consumer of the method. I would change the signature to `protected function debug($message)`